README.md

```diff
@@ -358,6 +358,11 @@ Convenience method to create a custom font based on a [fontello](http://fontello
 import { createIconSetFromFontello } from 'react-native-vector-icons';
 import fontelloConfig from './config.json';
 const Icon = createIconSetFromFontello(fontelloConfig);
+
+// LineAwesome Example
+import { createIconSetFromFontello } from 'react-native-vector-icons';
+import icoMoonConfig from './config.json';
```

Comment:
Do you mean fontelloConfig here? You're importing the Fontello function.

Reply:
nope. actual meant `createIconSetFromIcoMoon`. good catch 👍

Reply:
I removed this example and just included the explicit passed parameters to the `createIconSetFromFontello` method example. [commit](https://github.com/oblador/react-native-vector-icons/pull/818/commits/730e6d127d70d9d744a84358a0279a64a9a67853)